packages/core/src/render3/state.ts

```diff
@@ -129,31 +129,38 @@ export function getLView(): LView {
  * The reason why this value is `1` instead of `0` is because the `0`
  * value is reserved for the template.
  */
-const MIN_DIRECTIVE_ID = 1;
+let activeDirectiveId = 0;
 
-let activeDirectiveId = MIN_DIRECTIVE_ID;
+/**
+ * Flags used for an active element during change detection.
+ *
+ * These flags are used to signal whether to run any sub routine code when
```

Comment:
What does subroutine code mean here? Example would be helpful

```suggestion
 * These flags are used to signal whether to run any sub routine code when
```

Reply:
```
 * These flags are used within other instructions to inform cleanup or
 * exit operations to run when an element is being processed
```

Reply:
👍